ui/src/apiContext.ts

```diff
@@ -319,6 +319,26 @@ class FakeExportAPI {}
 
 class FakeUsersAPI {}
 
+const getAccessToken = () => {
```

Comment:
Use "function getAccessToken" for consistency

Reply:
Done